contracts/HubPool.sol

```diff
@@ -201,48 +214,101 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
 
     function initiateRelayerRefund(
         uint256[] memory bundleEvaluationBlockNumbers,
-        bytes32 poolRebalanceProof,
-        bytes32 destinationDistributionProof
+        uint64 poolRebalanceLeafCount,
+        bytes32 poolRebalanceRoot,
+        bytes32 destinationDistributionRoot
     ) public {
-        require(currentPendingRefundProposal == false, "Only one proposal at a time");
-        currentPendingRefundProposal = true;
-        relayerRefundRequests.push(
-            RelayerRefundRequest({
-                poolRebalanceProof: poolRebalanceProof,
-                destinationDistributionProof: destinationDistributionProof,
-                status: RefundRequestStatus.Pending
-            })
+        // The most recent refund proposal must be fully claimed before the next relayer refund bundle is initiated.
+        require(
+            relayerRefundRequests.length == 0 || // If this is the first initiated relay refund.
+                relayerRefundRequests[getNumberOfRelayRefunds()].unclaimedPoolRebalanceLeafs == 0,
+            "Last bundle has unclaimed leafs"
         );
 
+        uint64 requestExpirationTimestamp = uint64(getCurrentTime() + refundProposalLiveness);
+        uint256 relayerRefundId = relayerRefundRequests.length;
+        relayerRefundRequests.push();
+        relayerRefundRequests[relayerRefundId].requestExpirationTimestamp = requestExpirationTimestamp;
+        relayerRefundRequests[relayerRefundId].unclaimedPoolRebalanceLeafs = poolRebalanceLeafCount;
+        relayerRefundRequests[relayerRefundId].poolRebalanceRoot = poolRebalanceRoot;
+        relayerRefundRequests[relayerRefundId].destinationDistributionRoot = destinationDistributionRoot;
+        relayerRefundRequests[relayerRefundId].proposer = msg.sender;
+
```

Comment:
unfortunately this is the syntax we need to use because this struct contains a mapping for `mapping(uint256 => uint256) claimedBitMap;`. doing it like this leaves that mapping as uninitialized, giving it an empty mapping.